server/routerlicious/packages/tinylicious/src/services/tenantManager.ts

```diff
@@ -22,7 +28,19 @@ export class TinyliciousTenant implements ITenant {
 		private readonly url: string,
 		private readonly historianUrl: string,
 	) {
-		const historian = new Historian(historianUrl, false, false);
+		// Using an explicitly constructed rest wrapper so we can pass the Axios instance whose static defaults
+		// were modified by t9s, and avoid issues if the module that contains BasicRestWrapper depends on a different
+		// version of Axios.
+		const restWrapper = new BasicRestWrapper(
+			historianUrl,
+			undefined,
```

Comment:
Not for this PR, but how invasive would it be to update `BasicRestWrapper` to take a params object, rather than a long series of optional args?

Reply:
Not too hard immediately, I think; at first we just need to update usages in the server release group. But that package is actually also consumed by the client release group, so when we bump server there to take the new version, we'll have to update all usages on the client. However, it's technically a public API of @fluidframework/server-services-client, so we might want to stage it just to be safe.